python/sglang/srt/models/qwen2_moe.py

```diff
@@ -455,5 +458,14 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                     )
                     weight_loader(param, loaded_weight)
 
+        if self.torchao_config:
+            param_suffixes = set(["proj.weight"])
```

Comment:
1. Can it be one line?
```
apply_torch_ao(self, ...)
```

2. why do we still need `param_suffixes.union(set(entry[0] for entry in stacked_params_mapping))`. I did not check the code but I guess they also end with `proj.weight`?

Reply:
updated